android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -20,7 +20,10 @@
 import com.facebook.react.module.annotations.ReactModule;
 
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
```

Comment:
This does not work on Android - every version, use:
`private static final Charset UTF_8 = Charset.forName("UTF-8");` instead.